beacon-chain/core/statefeed/events.go

```diff
@@ -0,0 +1,45 @@
+package statefeed
+
+import "time"
+
+// How to add a new event to the feed:
+//   1. Add a constant describing the event to the list below
+//   2. Add a structure with the name `<event>Data` containing any data fields that should be supplied with the event.
+//
+// Note that the same event is supplied to all subscribers, so the event received by subscribers should be considered read-only.
+
+// EventType is the type that defines the type of event.
+type EventType int
+
+const (
+	// BlockProcessed is sent after a block has been processed and updated the state database.
+	BlockProcessed = iota + 1
+	// ChainStarted is sent when enough validators are active to start proposing blocks.
+	ChainStarted
+)
+
+// Event is the event that is sent with state feed updates
+type Event struct {
+	// Type is the type of event
+	Type EventType
+	// Data is event-specific data
+	Data interface{}
+}
+
+// BlockReceivedData is the data sent with BlockReceived events.
+type BlockReceivedData struct {
```

Comment:
We don't need this. Since this is unused and there's no real use case. Let's remove it for now

Reply:
Sorry yeah I removed the event but not the related data struct.  Gone now.